CruCentralCoast/Events/EventsVC.swift

```diff
@@ -23,11 +34,38 @@ class EventsVC: UITableViewController {
         self.tableView.estimatedRowHeight = 140
         
         DatabaseManager.instance.subscribeToDatabaseUpdates(self)
-        self.dataArray = DatabaseManager.instance.getEvents()
     }
-
+    
+    override func viewWillAppear(_ animated: Bool) {
+        let subscribedMovements = LocalStorage.preferences.getObject(forKey: .subscribedMovements) as? [String] ?? []
+        self.databaseArray = DatabaseManager.instance.getEvents()
```

Comment:
I believe you can use the "ANY" keyword in a predicate to filter this in a single line. Try "ANY movements.id IN %@". See https://nshipster.com/nspredicate/ and https://stackoverflow.com/questions/30076452/difference-between-all-any-and-some-operators-in-nspredicate for more info.

Reply:
That worked great!!!! Thank you!